test/extensions/dynamic_modules/test_data/rust/http_integration_test.rs

```diff
@@ -10,18 +10,18 @@ fn init() -> bool {
 fn new_http_filter_config_fn<EC: EnvoyHttpFilterConfig, EHF: EnvoyHttpFilter>(
   _envoy_filter_config: &mut EC,
   name: &str,
-  config: &str,
+  config: &[u8],
 ) -> Option<Box<dyn HttpFilterConfig<EC, EHF>>> {
   match name {
     "passthrough" => Some(Box::new(PassthroughHttpFilterConfig {})),
     "header_callbacks" => Some(Box::new(HeadersHttpFilterConfig {
-      headers_to_add: config.to_string(),
+      headers_to_add: String::from_utf8(config.to_owned()).unwrap_or_default(),
```

Comment:
can you panic instead of default as this is a test and i don't think non ut8 is not expected in the first place?